sys/compat/freebsd32/syscalls.master

```diff
@@ -142,18 +143,17 @@
 				   struct osigaction32 *nsa, \
 				   struct osigaction32 *osa); }
 47	AUE_GETGID	NOPROTO	{ gid_t getgid(void); }
-48	AUE_SIGPROCMASK	COMPAT	{ int freebsd32_sigprocmask(int how, \
-				   osigset_t mask); }
+48	AUE_SIGPROCMASK	COMPAT|NOPROTO	{ int sigprocmask(int how, osigset_t mask); }
 49	AUE_GETLOGIN	NOPROTO	{ int getlogin(char *namebuf, \
 				    u_int namelen); }
 50	AUE_SETLOGIN	NOPROTO	{ int setlogin(const char *namebuf); }
 51	AUE_ACCT	NOPROTO	{ int acct(const char *path); }
-52	AUE_SIGPENDING	COMPAT	{ int freebsd32_sigpending(void); }
+52	AUE_SIGPENDING	COMPAT|NOPROTO	{ int sigpending(void); }
 53	AUE_SIGALTSTACK	STD	{ int freebsd32_sigaltstack( \
-				    struct sigaltstack32 *ss, \
+				    const struct sigaltstack32 *ss, \
```

Comment:
This one doesn't seem to be const in !freebsd32, but it would seem reasonable to apply it there (and the kern_* helper); same for a couple more down below.